ngraph/test/backend/tan.in.cpp

```diff
@@ -34,7 +34,7 @@ using namespace ngraph;
 
 static string s_manifest = "${MANIFEST}";
 
-NGRAPH_TEST(${BACKEND_NAME}, tan)
+NGRAPH_TEST(${BACKEND_NAME}, tan_float)
```

Comment:
Could you remove tan backend test? And use only template reference tests instead?